metrics-exporter/base/deploymentconfig.yaml

```diff
@@ -168,7 +168,7 @@ spec:
                 configMapKeyRef:
                   name: prometheus
                   key: instance-metrics-exporter-infra
-            - name: METRICS_EXPORTER_INFRA_PROMETHEUS_INSTANCE
+            - name: MANAGEMENT_API_PROMETHEUS_INSTANCE
```

Comment:
seems like 'a' is missing in management-api in here: https://github.com/thoth-station/thoth-application/blob/80f1ccbce69ac15bbac034ed5c18e56b553cdbf0/metrics-exporter/base/deploymentconfig.yaml#L175
i missed this in previous pr review

Reply:
i guess its correct : https://github.com/thoth-station/thoth-application/blob/80f1ccbce69ac15bbac034ed5c18e56b553cdbf0/core/overlays/stage/configmaps.yaml#L52

Reply:
I updated it to be correct in English :D